### PR TITLE
Deploying Hyperscale Sql DB for PR/CI pipeline.

### PR DESCRIPTION
**Pull request by v-iyamauchi**
## Description
The PR is to deploy a Hyperscale Sql DB for PR/CI pipeline.

## Related issues
Addresses [issue #106036 and #111217].

[User Story 106036](https://microsofthealth.visualstudio.com/Health/_workitems/edit/106036): Provision HyperScale in OSS PR and CI pipelines and Run functional tests against it
[User Story 111217](https://microsofthealth.visualstudio.com/Health/_workitems/edit/111217): Add Hyperscale configurations to FHIR OSS custom deployment

## Testing
Tested in the PR/CI pipeline, and also in the custom deployment page on the azure portal.

## FHIR Team Checklist
- **Update the title** of the PR to be succinct and less than 65 characters
- **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- Tag the PR with the type of update: **Bug**, **Build**, **Dependencies**, **Enhancement**, **New-Feature** or **Documentation**
- Tag the PR with **Open source**, **Azure API for FHIR** (CosmosDB or common code) or **Azure Healthcare APIs** (SQL or common code) to specify where this change is intended to be released.
- Tag the PR with **Schema Version backward compatible** or **Schema Version backward incompatible** or **Schema Version unchanged** if this adds or updates Sql script which is/is not backward compatible with the code.
- [x] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

**Review by rbans96 — APPROVED** [review]

**Review by LTA-Thinking — DISMISSED** [review]

File: samples/templates/default-azuredeploy-docker.json [hunk 1]
```diff
@@ -27,20 +27,14 @@
         "appServicePlanSku": {
             "type": "string",
             "allowedValues": [
-                "F1",
-                "D1",
-                "B1",
-                "B2",
-                "B3",
```
Comment (LTA-Thinking) [inline samples/templates/default-azuredeploy-docker.json:34]: B3 is currently the value used in provision-deploy.yml. That file needs to be updated.

Comment (v-iyamauchi) [reply samples/templates/default-azuredeploy-docker.json:34]: I added B3 back to the list. PR build and deploy pipeline successfully completed.

**Review by LTA-Thinking — APPROVED** [review]

**Pull request merged**